setup.py

```diff
@@ -65,7 +65,7 @@
         # for kf
         'kfserving>=0.5.0',
         # for NED
-        'bootleg @ git+https://github.com/Mehrad0711/bootleg@b6c207259a83c21c77aeaf8088ee85f24ec7e708',
+        'bootleg @ git+https://github.com/Mehrad0711/bootleg@08da02ddc10540688747291f8ab54a1bacdfe433',
```

Comment:
I bumped bootleg to 1.0.5 which includes the newer version for pydantic so no changes will be needed here.